### PR TITLE
Jerry caption

### PR DESCRIPTION
**Pull request by JerryY1962**

### CODE REVIEW

File: _slides/03-JerryY1962.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to the house!"
 ---
 
-Welcome to the house of misfit kids!
+Welcome to the house of misfit kids!!!
```
Comment (PN-Washington) [inline _slides/03-JerryY1962.md]: Too many !!!!!

File: _slides/03-JerryY1962.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
 ---
 layout: slide
-title: "Welcome to our slide deck!"
+title: "Welcome to the house!"
```
Comment (frbaldhub) [inline _slides/03-JerryY1962.md:3]: Looks good.

File: _slides/03-JerryY1962.md (hunk 1 above)
Comment (PN-Washington) [inline _slides/03-JerryY1962.md]:
```suggestion
Welcome to the house of misfit kids!
```

File: _slides/03-JerryY1962.md [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
 ---
 
-Caption here
+Welcome to the house of misfit kids!!!
```
Comment (frbaldhub) [inline _slides/03-JerryY1962.md]: tr

Comment (frbaldhub) [reply _slides/03-JerryY1962.md]: done.

**Review by PN-Washington — CHANGES_REQUESTED** [review]
remove extra !

**Review by frbaldhub — COMMENTED** [review]
ready...

**Review by PN-Washington — APPROVED** [review]
Look good!

**Pull request merged**